packages/playwright/src/index.ts

```diff
@@ -185,8 +185,16 @@ export default class AxeBuilder {
 
   private async inject(frames: Frame[]): Promise<void> {
     for (const iframe of frames) {
-      await iframe.evaluate(await this.script());
-      await iframe.evaluate(await this.axeConfigure());
+      const race = new Promise((_, reject) => {
```

Comment:
Can you please extract this to its own PR? It's a `fix`, not a `chore`; it shouldn't be bundled into the "low risk" dep update.

(I don't think peer dep updates should ever be bundled into our "low risk" update group for that matter, but even if they were separate, this still needs a changelog entry)